vignettes/response-FDA IR-pilot3.qmd

```diff
@@ -0,0 +1,243 @@
+
+For the Pilot 3 project team, due to cross-industry collaboration, Linux was the best option for us to work in allowing us work in a stable R environment.
+
+1.  <https://cran.r-project.org/bin/windows/Rtools/rtools42/rtools.html>
+
+2.  <https://rconsortium.github.io/submissions-wg/minutes/2024-02-02/>
+
+3.  <https://www.quora.com/Is-the-R-programming-language-better-suited-for-windows-or-Linux>
+
+{{< pagebreak >}}
+
+[**Agency Comment 2**]{.underline}
+
+Differences between generated Pilot 2 ADaM and CDISC data sets : attributes, types (e.g. integer vs. double), and NA vs. NULL
+
+[**The R Consortium R Submission Working Group Response**]{.underline}
+
+The Pilot 3 project team explored all differences between the original CDISC datasets generated in SAS vs the Pilot 3 datasets generated in R. More detailed findings are documented in the ADRG in the section 'QC findings'. To address the agency's concerns, we explored these specific discrepancies that are present that could impact the analysis results.
+
+The team used three different packages:
+
+```         
+  library(arsenal)
+  library(diffdf)
+  library(waldo)
+  
+  path <- list(sdtm = "~/submissions-pilot3-adam/submission/sdtm",
+               adam = "~/submissions-pilot3-adam/submission/adam",
+               cdisc = "~/submissions-pilot3-adam/adam")
+```
+
+***ADSL***
+
+```         
+adsl <- read_xpt(file.path(path$adam, "adsl.xpt"))  
+adsl_cdisc <- read_xpt(file.path(path$cdisc, "adsl.xpt"))
+
+arsenal::comparedf(adsl, adsl_cdisc)
+diffdf::diffdf(adsl, adsl_cdisc, keys = c("STUDYID", "USUBJID"))
+waldo::compare(adsl, x_arg = "old", adsl_cdisc, y_arg = "new")
+
+
+`attr(old, 'label')` is a character vector ('Subject-Level Analysis Dataset')
+`attr(new, 'label')` is absent
+```
+{{< pagebreak >}}
+
+***ADTTE***
+
+```         
+adtte <- read_xpt(file.path(path$adam, "adtte.xpt"))
+adtte_cdisc <- read_xpt(file.path(path$cdisc, "adtte.xpt"))
+
+arsenal::comparedf(adtte, adtte_cdisc)
+diffdf::diffdf(adtte, adtte_cdisc, keys = c("STUDYID", "USUBJID")) 
+waldo::compare(adtte, x_arg = "old", adtte_cdisc, y_arg = "new")
+
+  ================================================
+   VARIABLE  ATTR_NAME   VALUES.BASE  VALUES.COMP 
+  ------------------------------------------------
+     AGE     format.sas     NULL           3      
+    AGEGR1   format.sas     NULL          $5      
+   AGEGR1N   format.sas     NULL           3      
+   EVNTDESC  format.sas     NULL          $25     
+    PARAM    format.sas     NULL          $32     
+   PARAMCD   format.sas     NULL          $4      
+     RACE    format.sas     NULL          $32     
+    RACEN    format.sas     NULL           3      
+    SAFFL    format.sas     NULL          $1      
+     SEX     format.sas     NULL          $1      
+  ------------------------------------------------
+```
+
+{{< pagebreak >}}
+
+***ADAE***
```

Comment:
Going back to the ADRG, this is explained in Appendix 2. 
![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/95468392/a8dc007b-7135-44fc-8eb3-85d5b9f5dc1d)

Maybe, @SHAESEN2 , it would be good to just copy the QC findings descriptions from the ADRG under each check shown here.  Maybe even paraphrased into one sentence.  Then at the end in the conclusion paragraph, you can add a note, "For further details, on each of these checks please refer to Appendix 2 QC Findings in the ADRG.". This could help clarify in this response letter.